### ISSUE TITLE
undefined method `pghero_pretty_ident'

### ISSUE THREAD

**Issue by oehlschl** [ankane/pghero#268]
We're running PgHero as a mounted rails engine, and after upgrading from Rails 4.2.10 to Rails 5.0, we started getting the following error in production:

```
ActionView::Template::Errorpg_hero/home#index
undefined method `pghero_pretty_ident' for ...
```

It looks like pghero_pretty_ident is defined in a BaseHelper file that's not explicitly included in the HomeController. And from what I just read, helper loading differs between Rails 4 and 5, and may differ by application with things like `config.action_controller.include_all_helpers`.

I'll play around a bit locally, but I think including that helper explicitly should resolve this issue.

**Comment by oehlschl** [comment]
Actually, just looked a bit further, and it seems like it might not be a version-specific issue at all.

That exception is being raised in the case of `@invalid_indexes` here, which may have only started happening for us recently in production.

https://github.com/ankane/pghero/blob/master/app/views/pg_hero/home/index.html.erb#L325

I was able to resolve the issue by explicitly including the helper locally, so I'll open a PR for that.

**Comment by ankane** [comment]
Hey @oehlschl, I think this issue is specific to your app, as PgHero has worked with Rails 5 for years...

**Comment by oehlschl** [comment]
Thanks @ankane -- I've updated the title of the issue. If you see in the second comment, I communicated that the behavior is instead specific to the use of pghero_pretty_ident when there are invalid indexes; since the behavior was recent, I likely mis-attributed it to Rails 5.

I'll see if I can't write a test case for this, but if you just take a look at the source, I'm not sure how that helper method would get included in the view since it's not autoloaded by Rails AFAIK.

**Comment by oehlschl** [comment]
@ankane just to confirm, I was able to replicate on a vanilla Rails 4 app by creating an invalid index. Again, I think it's just an uncommon case, which may have explained why it slipped under the radar (if indeed it did).

**Comment by ankane** [comment]
Hey @oehlschl, Rails automatically loads all helpers. I just installed PgHero on fresh Rails 5.0 and 5.2 apps with invalid indexes and `pghero_pretty_ident` works just fine.

**Comment by oehlschl** [comment]
Thanks @ankane -- confirmed here: https://api.rubyonrails.org/classes/ActionController/Helpers.html

Not sure how I missed the comment about eager helper loading, but I appreciate you testing Rails 5.0 and 5.2.

I've also confirmed that we've opted out of this behavior, as described in that article:
> In previous versions of Rails the controller will include a helper which matches the name of the controller [...]. To return old behavior set config.action_controller.include_all_helpers to false.

Therefore, it seems like PgHero will still be incompatible when mounted on any app that has `include_all_helpers` set to false, and that the best approach would be the explicit inclusion of that helper in the PR attached. Another alternative would be to name the helper so it's auto-magically loaded per the old behavior.

Worth noting that because `include_all_helpers` is still available in 5.2, I don't think it's a question of supporting old versions of Rails, and making one of these changes would make this gem more compatible with Rails installations at little cost.

Thoughts?
